next/content/docs/guides/vercel.md

```diff
@@ -1,73 +1,82 @@
-5. [Connect to the Neon database](#connect-to-the-neon-database)
+The Neon-Vercel integration connects your Vercel project to a Neon project. If you do not have a Neon project, you can create one when adding the integration. When adding the integration, you will select a database from the `main` branch of your Neon project. This will be your production database. Optionally, based on your selection, the integration creates a development branch that is branched from `main`, which you can use with your Vercel development environment.
 
-## Create a Neon project
+If you use [preview deployments](https://vercel.com/docs/concepts/deployments/preview-deployments) in Vercel, which are available by default for all Vercel projects, the integration automatically creates a database branch whenever a preview deployment is detected.
 
-If you do not have one already, create a Neon project. Save your connection details including your password. They are required when defining connection settings.
+The integration sets the following variables:
 
-1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
-2. Click **New Project**.
-3. Specify a name, a PostgreSQL version, a region, and click **Create Project**.
+- `PGHOST`
+- `PGUSER`
+- `PGDATABASE`
+- `PGPASSWORD`
+- `DATABASE_URL`
 
-For additional information about creating projects, see [Set up a project](/docs/get-started-with-neon/setting-up-a-project).
+The variables are set for your Vercel production environment, for your development environment if you chose to create a development branch, and for your preview environment whenever a preview deployment is detected.
 
-## Create a Next.js project
+## Prerequisites
 
-Create a Next.js project if you do not have one. For instructions, see [Create a Next.js App](https://nextjs.org/learn/basics/create-nextjs-app/setup), in the Vercel documentation.
+- A [Vercel account](https://vercel.com).
+- A Vercel project. If you do not have one, see [Creating a project](https://vercel.com/docs/concepts/projects/overview#creating-a-project), in the _Vercel documentation_.
 
-## Add a PostgreSQL client to your app
+## Add the Neon integration
 
-Add a PostgreSQL client to your app, such as `Postgres.js`. For instructions, refer to the [postgres.js Getting started](https://www.npmjs.com/package/postgres).
+1. Navigate to the [Neon Vercel integrations page](https://vercel.com/integrations/neon), and click **Add integration**.
+1. Select a Vercel account to add the integration to.
+1. Select the projects to which the integration will be added. You can select **All Projects** or a **Specific Project**.
+1. Review the permissions required by the integration, and click **Add Integration**.
+1. In the **Integrate Neon** dialog:
+    1. Select the Vercel project that you want to add the integration to. If the Vercel project is already connected to a Neon project, continuing the integration replaces the existing configuration.
```

Comment:
It actually most likely would fail, cause of existing env vars

Reply:
Removed.